### PR TITLE
Make 0.990.3 compliance 

### PR DESCRIPTION
**Pull request by ThisaruGuruge**

### CODE REVIEW

File: component/src/test/resources/services/kafka_service_stop.bal [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    string msg = "test_string";
+    byte[] byteMsg = msg.toByteArray("UTF-8");
+    var result = kafkaProducer->send(byteMsg, topic);
+}
+
+function funcKafkaGetResult() returns boolean {
+    return isSuccess;
+}
+
+
```
Comment (Shan1024) [inline component/src/test/resources/services/kafka_service_stop.bal]: Extra newline.

File: component/src/main/java/org/ballerinalang/kafka/nativeimpl/consumer/service/Stop.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import static org.ballerinalang.kafka.util.KafkaConstants.CONSUMER_SERVER_CONNECTOR_NAME;
+import static org.ballerinalang.kafka.util.KafkaConstants.CONSUMER_STRUCT_NAME;
+import static org.ballerinalang.kafka.util.KafkaConstants.FULL_PACKAGE_NAME;
+import static org.ballerinalang.kafka.util.KafkaConstants.KAFKA_NATIVE_PACKAGE;
+import static org.ballerinalang.kafka.util.KafkaConstants.ORG_NAME;
+
+/**
+ * Start server connector.
+ */
+
```
Comment (Shan1024) [inline component/src/main/java/org/ballerinalang/kafka/nativeimpl/consumer/service/Stop.java]: Extra newline.

File: component/src/main/resources/installation-scripts/README.txt [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+This guide contains instructions to install the Kafka ballerina module into your ballerina installation. This is a one-time task.
+Once done, you can then use this module across all your ballerina projects.
 
 In this directory, you can find two scripts of interest to you. Please note that when running the following scripts,
 you will need write permission to ballerina installation location.
 
-1. install.{sh/bat} - Installs the Kafka package.
-2. uninstall.{sh/bat} - Uninstalls the Kafka package if installed already.
+1. install.{sh/bat} - Installs the Kafka module.
+2. uninstall.{sh/bat} - Uninstalls the Kafka module if installed already.
```
Comment (Shan1024) [inline component/src/main/resources/installation-scripts/README.txt]:
```suggestion
2. uninstall.{sh/bat} - Uninstalls the Kafka module if it is already installed.
```

File: component/src/main/java/org/ballerinalang/kafka/nativeimpl/consumer/service/Stop.java (hunk 2 above)
Comment (ThisaruGuruge) [reply component/src/main/java/org/ballerinalang/kafka/nativeimpl/consumer/service/Stop.java]: Fixed

File: component/src/test/resources/services/kafka_service_stop.bal (hunk 1 above)
Comment (ThisaruGuruge) [reply component/src/test/resources/services/kafka_service_stop.bal]: Fixed.

**Pull request merged**